stats_test.go

```diff
@@ -41,15 +48,28 @@ func TestAddViewData(t *testing.T) {
 	}
 }
 
+func TestUDPExportError(t *testing.T) {
+	_, err := testExporter(Options{
+		StatsAddr: "invalid_address",
+	})
+
```

Comment:
Please remove this empty line or just write it all in one line (as you wish), for ex:
```go
if _, err := testExporter(Options{StatsAddr: "invalid_addr"}); err == nil {
    t.Fatal("expected an error")
}
```
(s/Errorf/Error)